website_event_filter_organizer/demo/event_demo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- event.event -->
+    <record id="event_organizer_0" model="event.event">
```

Comment:
Why adding this as demo instead on the `setUp` of the test?

Reply:
Ups!!!!!
As in V11 due to database cursor... Anyway I think that is valid :smile:

Reply:
Well, I prefer to not put data to be used in tests this way because they can be altered meanwhile, they add noise in real demo data, and any way, you haven't put them as noupdate="1".